src/common/contentPreference.ts

```diff
@@ -55,11 +55,13 @@ export const entityToNotificationTypeMap: Record<
 > = {
   [ContentPreferenceType.User]: [NotificationType.UserPostAdded],
   [ContentPreferenceType.Keyword]: [],
+  [ContentPreferenceType.FeedKeyword]: [],
```

Comment:
Was about to mention which one is which, but see you actually described it in comment.

> ContentPreferenceKeyword is used for my feed rows of feed_tag (feedId === userId)

Yeah think this is fine for me.